jekyll/_cci2/server/installation/phase-1-prerequisites.adoc

```diff
@@ -0,0 +1,809 @@
+
+endif::env-gcp[]
+
+ifndef::env-aws[]
+
+[#gke]
+=== GKE
+You can learn more about creating a GKE cluster https://cloud.google.com/kubernetes-engine/docs/how-to#creating-clusters[here].
+
+CAUTION: Do not use Autopilot cluster. CircleCI requires functionality that is not supported by GKE Autopilot.
+
+. https://cloud.google.com/sdk/gcloud[Install] and https://cloud.google.com/kubernetes-engine/docs/quickstart#defaults[configure] the GCP CLI for your GCP account. This includes creating a Google Project, which will be required to create a cluster within your project. 
++
+NOTE: When you create your project, make sure you also enable API access. If you do not enable API access, the command we will run next (to create your cluster) will fail.
++
+Setting the default `project id`, compute `zone` and `region` will make running subsequent commands easier:
++
+[source,shell]
+----
+gcloud config set project <PROJECT_ID>
+gcloud config set compute/zone <ZONE>
+gcloud config set compute/region <REGION>
+----
+. Create your cluster 
++
+TIP: CircleCI recommends using https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity[workload identities] to allow workloads/pods in your GKE clusters to impersonate Identity and Access Management (IAM) service accounts to access Google Cloud services. Use the following command to provision a simple cluster:
++
+[source,shell]
+----
+gcloud container clusters create circleci-server \
+  --num-nodes 3 \
+  --machine-type n1-standard-4 \
+  --workload-pool=<PROJECT_ID>.svc.id.goog
+----
++
+NOTE: Your kube-context should get updated with the new cluster credentials automatically.
++
+If you need to update your kube-context manually, you can by running the following:
++
+[source,shell]
+----
+gcloud container clusters get-credentials circleci-server
+----
+. Install the https://cloud.google.com/blog/products/containers-kubernetes/kubectl-auth-changes-in-gke[GKE authentication plugin] for `kubectl`:
++
+[source,shell]
+----
+gcloud components install gke-gcloud-auth-plugin
+----
+. Verify your cluster:
++
+[source,shell]
+----
+kubectl cluster-info
+----
+. Create a service account:
++
+[source,shell]
+----
+gcloud iam service-accounts create <SERVICE_ACCOUNT_ID> --description="<DESCRIPTION>" \
+  --display-name="<DISPLAY_NAME>"
+----
+. Retrieve credentials for the service account:
++
+[source,shell]
+----
+gcloud iam service-accounts keys create <KEY_FILE> \
+  --iam-account <SERVICE_ACCOUNT_ID>@<PROJECT_ID>.iam.gserviceaccount.com
+----
+endif::env-aws[]
+
+[#enable-workload-identities-in-gke]
+==== Enable Workload Identities in GKE (optional)
```

Comment:
I would like to move this section to another page, outside the flow of the installation guide if that will work. To avoid confusing people who have just followed the steps to set up their cluster. I'm wondering if we will potentially have customers who want to use an existing cluster without workload identities who will be following these steps and need this section at this point? I propose to make a page for this content in the operators guide and link to it at this point in the installation guide, something like: "If you already have a cluster you wish to use and want to enable workload identities, see the ***** guide." @circleci/server